wsd/ClientRequestDispatcher.cpp

```diff
@@ -802,11 +802,7 @@ void ClientRequestDispatcher::handleIncomingMessage(SocketDisposition& dispositi
                     launchAsyncCheckFileInfo(_id, accessDetails, RequestVettingStations,
                                              RvsHighWatermark);
                 }
-                servedSync = true;
             }
-
-            if (!servedSync)
```

Comment:
I think this "(!servedSync) test and setErrorAndShutdown" pair of lines should probably stay, but get moved to before line 789, so it appears at the end of the "if (pos != std::string::npos)" block so whatever used to happen still happens.

Reply:
Makes sense!